composer.json

```diff
@@ -27,7 +27,7 @@
         "toflar/psr6-symfony-http-cache-store": "^2.2 || ^3.0"
     },
     "require-dev": {
-        "ibexa/ci-scripts": "^0.1@dev",
+        "ibexa/ci-scripts": "dev-test-api-limits as 0.1.x-dev",
```

Comment:
Friendly reminder: this needs to be restored to the previous state before merging